.github/workflows/test-suite.yml

```diff
@@ -80,14 +80,6 @@ jobs:
     - name: Get latest version of stable Rust
       if: env.SELF_HOSTED_RUNNERS == false
       run: rustup update stable
-    - name: Use Node.js
-      uses: actions/setup-node@v2
-      with:
-        node-version: '14'
-    - name: Install windows build tools
-      run: |
-        choco install python visualstudio2019-workload-vctools -y
```

Comment:
We're also removing Python and VC tools here. Does this work because they're already preinstalled in the runner's Docker image?

Reply:
Good point - I'm not sure if this still works on the github hosted runner, I've just triggered a workflow on my repo.

Are `python` and `visualstudio2019-workload-vctools` used elsewhere? GitHub hosted windows runner also comes with python preinstalled, but be good to make sure our CI still works on GitHub runners.

Reply:
Build passed (except for the usual web3signer tests timeouts) on GitHub hosted runners too!
https://github.com/jimmygchen/lighthouse/actions/runs/6090881646

I think this is good to go!